src/main/java/com/avispl/symphony/dal/communicator/aver/ptz/AverPTZCommunicator.java

```diff
@@ -416,6 +412,11 @@ public void controlProperties(List<ControllableProperty> controllableProperties)
 			}
 		}
 
+		// Monitoring capabilities
+		populateMonitorCapabilities(stats);
```

Comment:
Why do you move this out of the try-catch block? I think the children methods of the populateMonitorCapabilities can throw the exception.